CHANGES.md

```diff
@@ -6,12 +6,20 @@ Version 3.9.0
 
 To be released.
 
+Due to changes in [#3529], a network ran with a prior version may not
+be compatible with this version,  specifically, those that ran with
+[Libplanet 2.0.0] and onwards prior to this release that have included
+`Transaction`s that aren't compatible with the updated specification in [#3529].
```

Comment:
Should be placed at Backward incompatible changes...?

Reply:
Eh, thought it was important enough to be put in the header section. Not sure where exactly to put it.